vision_statement.md

```diff
@@ -112,22 +112,13 @@ conduct](https://www.python.org/psf/codeofconduct/).
 
 ## Organizational structure
 
-The Coordinating Committee (CC) will oversee the PlasmaPy project.
-This committee will take a big picture approach defining the goals and
-future directions of PlasmaPy.  The CC will make sure that roles are
-being filled, necessary tasks are being completed, facilitating
-intergroup communication (including the email list and HipChat group),
-coordinating and delegating tasks, seeking funding mechanisms,
-facilitating compromises and cooperation, and fostering a culture of
-appreciation.
- 
-The Code Development Committee (CDC) will directly oversee the
-development of the codebase.  This committee will be responsible for
-managing the project repository, reviewing and accepting pull requests,
-ensuring intercompatibility between different subpackages, and
-coordinating automated testing.  The CDC will be responsible for
-ensuring that the PlasmaPy codebase adheres to the standards defined
-in this document.
+The Coordinating Committee (CC) will oversee the PlasmaPy project and
+code development.  The CC will ensure that roles are being filled,
+facilitate intergroup communication, coordinate and delegate tasks,
```

Comment:
What groups are intercommunicating if we're cutting down to one committee?

Reply:
Good catch - I changed it to "facilitate community-wide communication".